pkgs/development/interpreters/python/wrap.sh

```diff
@@ -47,7 +47,7 @@ wrapPythonProgramsIn() {
     buildPythonPath "$pythonPath"
 
     # Find all regular files in the output directory that are executable.
-    for f in $(find "$dir" -type f -perm -0100); do
+    find "$dir" -type f -perm -0100 | while read f; do
```

Comment:
I've built and manually inspected `octoprint`. I assume it would fail to make any wrappers if done incorrectly -- a wrapper was there and seemed fine.